src/actions/bloodlinesActions.js

```diff
@@ -1,307 +1,120 @@
+export const HANDLE_PAGED = "HANDLE_PAGED";
+export const ERROR_PAGED = "ERROR_PAGED";
+export const TRIGGERS = "TRIGGERS";
+export const CONTENTS = "CONTENTS";
 
-export const GET_ALL_CONTENT = "GET_ALL_CONTENT";
-export const HANDLE_GET_ALL_CONTENT = "HANDLE_GET_ALL_CONTENT";
-export const ERROR_GET_ALL_CONTENT = "ERROR_GET_ALL_CONTENT";
+export const REQUEST = "REQUEST";
+export const HANDLE = "HANDLE";
+export const ERROR = "ERROR";
+export const TIMEOUT = "TIMEOUT";
```

Comment:
I've looked around and it seems we could do something like this:

(NEW) `actionTypes.js`
```
export default {
  FETCH_ME: 'FETCH_ME',
  FETCH_LIKES: 'FETCH_LIKES',
  FETCH_TRACKS: 'FETCH_TRACKS',
  FETCH_USER: 'FETCH USER'
};
```

`bloodlinesActions.js`
```
import ActionTypes from './actionTypes';
...
ActionTypes.ERROR
...
```

`bloodlinesReducer.js`
```
import ActionTypes from './actionTypes';
...
ActionTypes.ERROR
...
```

Would you be interested in adding this here?  We can do it in a follow-up if you want too!  I think it reduces boiler plate on the Reducer end for imports and consolidates all the constants for consistent action naming.